public/app.prod.json

```diff
@@ -20,5 +20,6 @@
         "company": "ScottLogic",
         "description": "Hosts BitFlux as an OpenFin application.",
         "name": "StockFlux"
-    }
+    },
+    "splashScreenImage": "http://scottlogic.github.io/StockFlux/master/assets/png/splashscreen.png"
```

Comment:
Thinking about this more... I think we should be using the same URL as above, for consistency (i.e. `http://localhost:5001`). I'm not sure using the deployed version makes much sense here.

Obviously, we'll have to make sure we have appropriate configs for the deployed version(s) too, but let's deal with that separately.